### ISSUE TITLE
"No Such Process" on OS X 10.8.4

### ISSUE THREAD

**Issue by reduxd** [malcolmstill/thunner#12]
Trying to choose any of the options on the menu results in the program crashing and displaying the following error. Any tips?

![screen shot 2013-08-15 at 12 29 43 am](https://f.cloud.github.com/assets/1561087/967177/cd5b73e0-056b-11e3-9150-bcbee9e3a34a.png)

**Comment by reduxd** [comment]
Also reproducable in Ubuntu 13.04

![screen shot 2013-08-15 at 1 00 11 am](https://f.cloud.github.com/assets/1561087/967261/286a9b9a-0570-11e3-87cf-b63468fa3f78.png)

**Comment by reduxd** [comment]
mplayer is indeed installed on both OS X and Ubuntu. On Ubuntu, mplayer was
installed using apt-get, and on OS X it was symlinked to the binary inside
mplayer extended.app

**Comment by malcolmstill** [comment]
Can you just confirm that you have mplayer installed? Also if you have the latest gmusicapi installed via pip install gmusicapi you'll need the issue11 branch.

**Comment by malcolmstill** [comment]
Sorry, I deleted the previous comment after re-reading your comments.

**Comment by reduxd** [comment]
Yep, got the issue11 branch after reading through issue11 in order to have the latest version.

**Comment by malcolmstill** [comment]
Right it seems I've not updated the issue11 branch for gmusicapi 2.0.0 (I'm still calling get_stream_url rather than get_stream_urls). Let me update that and see if it does anything for you.

**Comment by malcolmstill** [comment]
I've just updated issue11; can you try it out?

**Comment by reduxd** [comment]
I replaced the lines myself after noticing it errored out on that issue.
Sorry for not mentioning it before.

**Comment by malcolmstill** [comment]
So you're still getting the error after having amended those lines? Can you give me the version of everything you're running (python, gmusicapi, mplayer etc.)?

**Comment by reduxd** [comment]
Python 2.7.5, gmusicapi 2.0.0, mplayer 1.1.1 (according to the mplayer
extended site)

**Comment by malcolmstill** [comment]
Let me look at this tomorrow (it's after 5:00 am here).

**Comment by reduxd** [comment]
Pulled the latest version and it worked with your get_stream_urls changes. I think my changes screwed up something that made it crash.
